.github/workflows/intel.yml

```diff
@@ -14,7 +14,7 @@ defaults:
 
 # Set I_MPI_CC/F90 so Intel MPI wrapper uses icc/ifort instead of gcc/gfortran
```

Comment:
@aliabdolali from the instructions at https://github.com/NOAA-EMC/hpc-stack/issues/501, the Intel build relies on `gcc` for some functionality.  So instead of `icc`/`ifort`, would you not want this section to select `gcc`/`ifort`?